vignettes/simmer-04-bank-2.Rmd

```diff
@@ -406,7 +406,7 @@ door_schedule <- schedule(c(0, openTime), c(0, Inf))
 doorman <-
   trajectory() %>%
   timeout(openTime) %>%
-  log_("Ladies and Gentlemen! You man all enter.")
+  log_("Ladies and Gentlemen! You can all enter.")
```

Comment:
Haha, you're right!  Too late now ;-)
![temp](https://user-images.githubusercontent.com/3522552/27886304-6440830a-61d3-11e7-8d62-b26fdd35c0a7.png)